webtool/test_transcribe.py

```
@@ -2312,3 +2314,163 @@ def korrektur_wie_echt(project, b, **kw):
+
+
+def test_die_wache_sieht_auch_das_programm_aus_executable():
+    """`executable=` ersetzt das Programm aus `args[0]` — ohne die Zeile in der Wache startet
+    `Popen(["egal", "install", "x"], executable=".../pip")` echtes pip, weil `args` das Wort
+    nicht traegt. Der Pfad hier existiert nicht: ohne Wache endet der Aufruf in
+    `FileNotFoundError`, mit Wache in `pytest.fail` VOR dem Start (CodeRabbit-Bot, Major,
+    PR #529)."""
+    import subprocess as sp
+
+    with pytest.raises(pytest.fail.Exception):
+        sp.Popen(["egal", "install", "x"], executable="/nirgends/pip")
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)